src/components/testimonial/Testimonial.css

```diff
@@ -0,0 +1,121 @@
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: .3rem;
+  flex-basis: 20rem;
+  padding: 1rem;
+  max-width: 20rem;
+  flex-grow: 1;
+  row-gap: .7rem;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  border-bottom: 1px solid;
+
+  & img {
+    position: relative;
+    width: 100%;
+    border-radius: 1rem;
+  }
+
+  .image-container {
+    position: relative;
+    height: 10rem;
+    background-image: url("../../assets/britnie.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    border-radius: 1rem;
+
+    & .button-box {
+      position: relative;
+      border: 2px solid;
+
+
+      content: '';
+      background: #FF5555;
+      border-radius: 20px;
+      position: absolute;
+      width: 45px;
+      height: 45px;
+      /* Centering the pseudo-element */
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%) matrix(0.24, 0.96, -0.98, 0.22, 0, 0);
+      z-index: 3;
+    }
+
+    &::after {
+      content: '';
+      width: 0;
+      height: 0;
+      border-width: 10px 10px 10px 15px;
+      border-style: solid;
+      border-color: transparent transparent transparent white;
+      margin-top: 1.5rem;
+      margin-left: .5rem;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      z-index: 4;
+      position: absolute;
+
+    }
+  }
+
+    &::before {
+
+  }
+
+  .link-box {
+    display: flex;
+    margin-left: 2rem;
+
+
+
+
+    & span {
+      width: 1rem;
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  & a {
+    position: relative;
+    text-align: center;
+    color: black;
+    text-decoration: none;
+    font-weight: 500;
+    padding: 0 .2rem;
+  }
+
+  & a::before {

```

Comment:
Je gebruikt nu deze pseudo-class om de rode cirkel te maken, maar omdat de cirkel echt een "knop" moet voorstellen, kun je je afvragen of dat wel een pseudo-element moet zijn. Misschien kun je de knop beter zien als een span met een achtergrondkleur, waarop je dan de witte driehoek als pseudo-element plaatst. Dit zorgt ervoor dat je de witte driehoek altijd in het midden van de rode cirkel kunt zetten (absoluut ten opzichte van de cirkel), ongeacht hoe groot de cirkel is (of wordt). 

En, gezien hij in meerdere maten terugkomt, zou je er zelf een componentje van kunnen maken!